routing/payment_lifecycle_test.go

```diff
@@ -425,280 +417,6 @@ func TestRouterPaymentStateMachine(t *testing.T) {
 			routes:     []*route.Route{rt},
 			paymentErr: channeldb.FailureReasonNoRoute,
 		},
-
-		// =====================================
```

Comment:
these the same tests that are later added back?

Reply:
Yep, and I've put them into the same commit now.